extra-utils/throttled/autobuild/build

```diff
@@ -1,9 +1,32 @@
-abinfo "Compiling python sources..."
-python3 -m compileall *.py
-abinfo "Installing sutff..."
-install -Dm644 etc/lenovo_fix.conf "$PKGDIR"/etc/lenovo_fix.conf
-install -Dm644 systemd/lenovo_fix.service "$PKGDIR"/usr/lib/systemd/system/lenovo_fix.service
-install -Dm755 lenovo_fix.py "$PKGDIR"/usr/lib/$PKGNAME/lenovo_fix.py
-install -Dm755 mmio.py "$PKGDIR"/usr/lib/$PKGNAME/mmio.py
-cp -a __pycache__ "$PKGDIR"/usr/lib/$PKGNAME/
-install -Dm644 LICENSE "$PKGDIR"/usr/share/licenses/$PKGNAME/LICENSE
+abinfo "Installing scripts and byte codes..."
+mkdir -pv "$PKGDIR"/usr/lib/python"$ABPY3VER"/site-package/throttled 
+install -Dvm755 "$SRCDIR"/throttled.py \
+    "$PKGDIR"/usr/lib/python"$ABPY3VER"/site-package/throttled/throttled.py
+install -Dvm755 "$SRCDIR"/mmio.py \
+    "$PKGDIR"/usr/lib/python"$ABPY3VER"/site-package/throttled/mmio.py
+install -Dvm644 "$SRCDIR"/etc/throttled.conf \
+    "$PKGDIR"/etc/throttled.conf
+install -Dvm644 "$SRCDIR"/systemd/throttled.service \
+    "$PKGDIR"/usr/lib/systemd/system/throttled.service
+cp -va "$SRCDIR"/__pycache__ \
+    "$PKGDIR"/usr/lib/python"$ABPY3VER"/site-package/throttled/
+
+mkdir -pv "$PKGDIR"/usr/bin/
+ln -vs /usr/lib/python"$ABPY3VER"/site-package/throttled/throttled.py \
+    "$PKGDIR"/usr/bin/throttled 
+
+abinfo "Installing LICENSE and README ..."
+install -Dvm644 LICENSE \
```

Comment:
Do not install licenses manually, Autobuild3 will handle this automatically. By the way, our license file location is...

`/usr/share/doc/$PKGNAME/`